solver/llbsolver/solver.go

```diff
@@ -7,6 +7,8 @@ import (
 	"strings"
 	"time"
 
+	"golang.org/x/sync/errgroup"
+
```

Comment:
Get your editor under control! Rearranging imports in a cherry-pick needlessly increases the size of the diff, making it harder to review.